docs/structure-of-a-contract.rst

```diff
@@ -210,16 +211,17 @@ Tags
 
 All tags are optional. The following table explains the purpose of each NatSpec tag and where it may be used:
 
-=========== ======================================== ==================
-Tag         Description                              Context
-=========== ======================================== ==================
-``@title``  Title that describes the contract        contract
-``@author`` Name of the author                       contract, function
-``@notice`` Explain to an end user what this does    contract, function
-``@dev``    Explain to a developer any extra details contract, function
-``@param``  Documents a single parameter             function
-``@return`` Documents one or all return variable(s)  function
-=========== ======================================== ==================
+===========  ======================================== ==================
```

Comment:
leaving 2 blank spaces in between `=` causes the table not to render.

Reply:
```suggestion
============ ======================================== ==================
```

Reply:
Yeah, has to be on all lines.